plugins/bulk-import/src/hooks/useRepositories.test.ts

```diff
@@ -10,12 +9,18 @@ jest.mock('@backstage/core-plugin-api', () => ({
   useApi: jest.fn(),
 }));
 
-const mockUseApi = useApi as jest.MockedFunction<typeof useApi>;
+jest.mock('@tanstack/react-query', () => ({
+  ...jest.requireActual('@tanstack/react-query'),
+  useQuery: jest.fn(),
+}));
 
 describe('useRepositories', () => {
   it('should return repositories', async () => {
-    mockUseApi.mockReturnValue({
-      dataFetcher: jest.fn().mockReturnValue(mockGetRepositories),
+    (useQuery as jest.Mock).mockReturnValue({
+      data: mockGetRepositories,
+      isLoading: false,
+      error: '',
```

Comment:
I would expect error is null when there is no error.

```suggestion
      error: null,
```